airqo-platform/AirQo-frontend
# [NetManger] Improve device-registry user experience

#### Summary of Changes (What does this PR do?)
- Add deploy status on device-registry tables
- Add deploy status, time elapse checks and field check on deploy status
- Update maintenance logs to include `description/tags`

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Install requirements `npm install`
* Start app `npm run stage-mac` (macOS) or `npm run stage-pc` (windows platform)
* Please check the screenshot sections for the changes to look out for.

#### What are the relevant tickets?

#### Screenshots (optional)
- Deploy status on device-registry table
![image](https://user-images.githubusercontent.com/39955305/99841510-56adbe00-2b7f-11eb-9875-f06e7cc4a658.png)

- deploy status, time elapse checks and field check on deploy status
![image](https://user-images.githubusercontent.com/39955305/99841851-e0f62200-2b7f-11eb-95ce-82431b7c857e.png)
